localstack/config.py

```diff
@@ -1332,12 +1312,7 @@ def get_edge_url(localstack_hostname=None, protocol=None):
 
 
 def edge_ports_info():
-    if EDGE_PORT_HTTP:
-        result = "ports %s/%s" % (EDGE_PORT, EDGE_PORT_HTTP)
-    else:
-        result = "port %s" % EDGE_PORT
-    result = "%s %s" % (get_protocol(), result)
-    return result
+    return f"{get_protocol()} port {GATEWAY_LISTEN[0].port}"
```

Comment:
Not sure what this method exactly is for, but shouldn't we print all ports of GATEWAY_LISTEN here, not only the first?

Reply:
It seems used for log messages, hence adding all ports makes sense.
Done 👍